build_dashboard.sh

```diff
@@ -1,3 +1,14 @@
-cd `dirname $0`/app/dashboard
-VITE_BASE_API=/api/ npm run build --if-present -- --outDir build --base '/dashboard/'
-cp ./build/index.html ./build/404.html
\ No newline at end of file
+#!/usr/bin/env bash
```

Comment:
Is it better to get path as input instead of reading env file again.

Reply:
Hmm, that's a good idea💡